packages/components/src/progress-bar/index.tsx

```diff
@@ -26,8 +26,12 @@ function UnforwardedProgressBar(
 	return (
 		<ProgressBarStyled.Track className={ className }>
 			<ProgressBarStyled.Indicator
+				style={
+					{
+						'--indicator-width': `${ value }%`,
+					} as CSSProperties
+				}
```

Comment:
Do we even need a CSS var? Can't we just set the `width` directly?

```suggestion
				style={ {
					width: `${ value }%`,
				} }
```

That would allow us to avoid the typecasting too.

Reply:
See https://github.com/WordPress/gutenberg/pull/60388#issuecomment-2034199693 regarding this.

The typecasting is definitely unfortunate, and something that people have tried to fix in the upstream types - hopefully it'll be solved at some point. I don't think that a minor typing limitation should prevent us from using a core CSS feature though :P

Reply:
It's totally fine to keep it. I was nitpicking 😉